communication/socket/create-raw-socket-in-windows.yml

```diff
@@ -0,0 +1,23 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: create raw socket in Windows
```

Comment:
similar socket rules are OS independent, I'd suggest to do the same here

Reply:
Thanks for the suggestion! Renamed file and implementing the additional feedback.